dbhelpers/helpers.js

```diff
@@ -0,0 +1,41 @@
+const knex = require("knex");
+const knexConfig = require("../knexfile.js");
+const db = knex(knexConfig.development);
+
+module.exports = {
+  get: (id,which) => {
```

Comment:
Great job with the `get` and a **dynamic** _which_ param. Very cool. Same with your `delete`, `insert`, and `edit`.